src/Validation/DonationValidator.php

```diff
@@ -27,13 +28,16 @@ public function __construct( AmountValidator $amountValidator,
 								 PersonNameValidator $nameValidator,
 								 PhysicalAddressValidator $addressValidator,
 								 AllowedValuesValidator $paymentTypeValidator,
+								 BankDataValidator $bankDataValidator,
 								 MailValidator $mailValidator ) {
+		// TODO: inject a ValidatorObject to reduce number of parameters
```

Comment:
I'm unhappy about how the list of expected parameters to create the use case object increases and thought about injecting a single container for all those validators.